package com.campusdual.classroom;

public class Exercise29 {
    public static void main(String[] args) {
        try {
            // Lanzar una excepci贸n de tipo RuntimeException
            throw new RuntimeException("Este es un lanzamiento intencionado de una excepci贸n.");
        } catch (RuntimeException e) {
            // Controlar la excepci贸n para evitar que el programa termine abruptamente
            System.out.println("Excepci贸n controlada: " + e.getMessage());
        }
    }
}
